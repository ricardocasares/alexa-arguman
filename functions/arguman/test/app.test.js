const arguman = require("../app");
const request = require("./requests/launch");

describe("arguman", () => {
  test("should launch properly", () => {
    return expect(arguman.request(request)).resolves.toMatchSnapshot();
  });
});
