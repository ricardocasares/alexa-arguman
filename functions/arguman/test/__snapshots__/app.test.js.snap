// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`arguman should launch properly 1`] = `
Object {
  "response": Object {
    "directives": Array [],
    "outputSpeech": Object {
      "ssml": "<speak>Welcome to Arguman!</speak>",
      "type": "SSML",
    },
    "shouldEndSession": false,
  },
  "sessionAttributes": Object {},
  "version": "1.0",
}
`;
